src/components/basin-analyzer/component.jsx

```diff
@@ -60,7 +60,7 @@ class BasinAnalyzer extends PureComponent {
             <span className="title">Analyze</span>
             { points.length > 0 &&
               <span className="">
-                Location: [{points[0].lat}, {points[0].lng}]
+                Location: {points.length > 1 ? <React.Fragment>[{points.length} Locations]</React.Fragment> : <React.Fragment>[{points[0].lat}, {points[0].lng}]</React.Fragment>}
```

Comment:
👍 Nice improvement.
I think the outer braces ([]) could be removed here for the lay user, but that's just me.

Reply:
Agreed.  I kept them so as to keep it consistent